hackathon_site/dashboard/frontend/src/components/general/Navbar/Navbar.module.scss

```diff
@@ -0,0 +1,43 @@
+@import './../../../assets/abstracts/mixins';
+@import './../../../assets/abstracts/variables';
+
+.nav {
+	color: color(white);
+	background-color: color(secondary) !important;
+	
+	&Toolbar {
+		justify-content: space-between;
+	}
+
+	&Logo {
+		display: flex;
+
+		&Img {
+			height: 30px;
+		}
+	}
+
+	button {
+		border-radius: 0;
+
+		&:hover {
+			background-color: rgba(355, 355, 355, 0.2);
+		}
+	}
+
+	&Icons {
+		align-self: right;
```

Comment:
`right` isn't a valid property value for `align-self`: https://www.w3schools.com/cssref/css3_pr_align-self.asp

Reply:
oh my bad i forgot to delete thx